app/org/sagebionetworks/bridge/dynamodb/DynamoSubpopulationDao.java

```diff
@@ -91,6 +94,12 @@ public Subpopulation updateSubpopulation(Subpopulation subpop) {
         DynamoDBQueryExpression<DynamoSubpopulation> query = 
                 new DynamoDBQueryExpression<DynamoSubpopulation>().withHashKeyValues(hashKey);
         
+        if (!includeDeleted) {
+            query.withQueryFilterEntry("deleted", new Condition()
+               .withComparisonOperator(ComparisonOperator.EQ)
+               .withAttributeValueList(new AttributeValue().withN("0")));
+        }
+
```

Comment:
Reverted filtering and added test of retrieving deleted as part of all study subpopulations.